### PR TITLE
Changelog for 1.13.0

### PR DESCRIPTION
**Pull request by straight-shoota**
Preview: https://github.com/crystal-lang/crystal/blob/changelog/1.13.0/CHANGELOG.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,367 @@
...
+
+[#14463]: https://github.com/crystal-lang/crystal/pull/14463
+[#14393]: https://github.com/crystal-lang/crystal/pull/14393
+[#14553]: https://github.com/crystal-lang/crystal/pull/14553
+
+#### tools
+
+- *(docs-generator)* **[security]** Sanitize doc generation ([#13786], thanks @nobodywasishere)
+
+[#13786]: https://github.com/crystal-lang/crystal/pull/13786
```
Comment (Sija) [inline CHANGELOG.md:77]: Including this seems confusing, since it was reverted.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,367 @@
...
+
+- *(codegen)* **[breaking]** Remove `CRYSTAL_LIBRARY_RPATH` and delay-load helper ([#14598], thanks @HertzDevil)
+
+[#14598]: https://github.com/crystal-lang/crystal/pull/14598
+
+#### tools
+
+- *(docs-generator)* Revert "Sanitize doc generation" ([#14595], thanks @straight-shoota)
+
+[#14595]: https://github.com/crystal-lang/crystal/pull/14595
```
Comment (Sija) [inline CHANGELOG.md:197]: ditto

File: CHANGELOG.md (hunk 1 above)
Comment (straight-shoota) [reply CHANGELOG.md:77]: Yes, I have already removed it from the milestone, but the changelog needs to be regenerated.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,427 @@
...
+
+#### stdlib
+
+- *(collection)* Add `Array#insert_all` ([#14486], thanks @summer-alice)
+- *(collection)* Improve compile time error for `#sort(&block : T, T -> U)` ([#14693], thanks @beta-ziliani)
+- *(concurrency)* Add `WaitGroup` synchronization primitive ([#14167], thanks @ysbaddaden)
+- *(concurrency)* Add `Thread.new` yields itself ([#14543], thanks @ysbaddaden)
+- *(concurrency)* Add support for `Atomic(Bool)` ([#14532], thanks @ysbaddaden)
+- *(concurrency)* Add `Thread.sleep(Time::Span)` ([#14715], thanks @ysbaddaden)
+- *(files)* **[deprecation]** Move `File.readable?`, `.writable?`, `.executable?` to `File::Info` ([#14484], thanks @straight-shoota)
```
Comment (beta-ziliani) [inline CHANGELOG.md:25]: This isn't a feature

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,427 @@
...
+- *(files)* Implement `IO#tty?` in Win32 ([#14421], thanks @HertzDevil)
+- *(files)* Implement `File.readable?` and `.writable?` in Win32 ([#14420], thanks @HertzDevil)
+- *(files)* Make `File.readable?` and `.writable?` follow symlinks on Windows ([#14514], thanks @HertzDevil)
+- *(llvm)* Add some missing `LLVM::Context` bindings ([#14612], thanks @ysbaddaden)
+- *(llvm)* Do not strip the macOS target triple ([#14466], thanks @hovsater)
```
Comment (beta-ziliani) [inline CHANGELOG.md:30]: Maybe a bugfix?

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,427 @@
...
+
+[#14757]: https://github.com/crystal-lang/crystal/pull/14757
+[#14719]: https://github.com/crystal-lang/crystal/pull/14719
+[#14545]: https://github.com/crystal-lang/crystal/pull/14545
+
+### Infrastructure
+
+- *(interpreter)* Run `primitives_spec` with the interpreter on CI ([#14438], thanks @HertzDevil)
+
+[#14438]: https://github.com/crystal-lang/crystal/pull/14438
```
Comment (beta-ziliani) [inline CHANGELOG.md:385]: This should be moved to the end

File: CHANGELOG.md (hunk 4 above)
Comment (straight-shoota) [reply CHANGELOG.md:30]: Depends on perspective. I understand it as a feature enhancement of the LLVM library to include the deployment version into the default target triple.

File: CHANGELOG.md (hunk 5 above)
Comment (beta-ziliani) [reply CHANGELOG.md:385]: but why isn't it reflected here still?

Comment (straight-shoota) [reply CHANGELOG.md:385]: I hadn't regenerated the changelog since merging that PR.

Comment (beta-ziliani) [reply CHANGELOG.md]: ah! sorry for the noise then!

**Review by ysbaddaden — APPROVED** [review]

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,445 @@
...
+- *(log)* Support `UInt32` and `UInt64` in Log Context ([#14459], thanks @toddsundsted)
+- *(macros)* Add AST node methods for macro-related nodes ([#14492], thanks @HertzDevil)
+- *(macros)* Support short blocks in `#[]` operator call's macro interpolation ([#14523], thanks @HertzDevil)
+- *(macros)* Add macro methods for `Select` ([#14600], thanks @HertzDevil)
+- *(macros)* Add `TypeNode#private?`, `#public?` and `#visibility` ([#11696], thanks @Hadeweka)
+- *(macros)* Add `StringLiteral#to_utf16` ([#14676], thanks @ysbaddaden)
+- *(networking)* Relax type restriction of handlers in `HTTP::Server.new` to `Indexable(HTTP::Handler)` ([#14413], thanks @hugopl)
+- *(networking)* **[security]** OpenSSL: don't change default cipher suites ([#14655], thanks @ysbaddaden)
+- *(networking)* Allow parsing cookies with space in the value ([#14455], thanks @anton7c3)
+- *(runtime)* Allow `Atomic`s of pointer types ([#14401], thanks @HertzDevil)
```
Comment (ysbaddaden) [inline CHANGELOG.md:39]: **question**: shouldn't `Atomic(Bool)` and `Atomic(Pointer)` be grouped together? That is both be `concurrency` or `runtime`.

**Review by ysbaddaden — APPROVED** [review]

**Review by beta-ziliani — APPROVED** [review]

**Pull request merged**